component/main.jsonnet

```
@@ -27,6 +27,18 @@ local dockercfg = std.trace(
 
 local motd = import 'motd.libsonnet';
 
+local caBundle = {
+  apiVersion: 'v1',
+  kind: 'ConfigMap',
+  metadata: {
+    name: 'custom-ca-bundle',
```

Might make sense to use a clearer Project Syn prefix for the name, e.g.

```suggestion
    name: 'syn-ca-bundle',
```